coursedata/level-defaults/nl.yaml

```diff
@@ -28,8 +28,8 @@
             {echo} dus je lievelingskleur is
     -   name: "{print} emojis"
         explanation: "print een emoji met `{print}`"
-        example: "Example: {print} 🙋‍ 🌍 ❗"
-        demo_code: "{print} 🙋 🌍 ❗"
+        example: "Example: {print} \U0001F64B‍ \U0001F30D ❗"
+        demo_code: "{print} \U0001F64B \U0001F30D ❗"
```

Comment:
Interessant, emoji komen niet goed mee terug uit Weblate?

Reply:
Interessant? Irritant :-)

Reply:
Komt deze Unicode wel weer goed in de website?

Reply:
Found this in the Weblate 4.2 documentation:

![Schermafbeelding 2022-02-24 om 14 39 55](https://user-images.githubusercontent.com/38155424/155535114-c895dcdd-c1e7-4e06-96bb-732b48b63930.png)

https://docs.weblate.org/_/downloads/en/weblate-4.2/pdf/ (page 175)

Reply:
ik weet niet of dit goed gaat op de site, kan jij deze branch eens lokaal testen @TiBiBa (of @Mark-Giesen zelf?)

Reply:
Ah goede catch @TiBiBa! Dat kunnen we ff instellen op weblate

Reply:
Ik weet niet zeker of ik deze branch lokaal kan testen omdat het van een remote repo is. Kan hem in ieder geval niet vinden bij 'branches'.

Reply:
hmmm interesting! bij mij staat het gewoon bij PR's in github desktop:

![image](https://user-images.githubusercontent.com/1003685/155543978-0f9c3485-b09d-44bd-8e7c-d63dbae791b3.png)

Reply:
Het werkt nog! Yamls zijn natuurlijk wel wat minder leesbaar, maar als we die toch niet meer als text aanpassen is dat geen probleem:

![Schermafbeelding 2022-02-24 om 15 39 07](https://user-images.githubusercontent.com/38155424/155545526-5a205b79-fe5a-4aed-a749-da6bf389f576.png)